scripts/hid_configurator/configurator_core.py

```diff
@@ -326,6 +405,17 @@ def get_device_type(pid):
             return device_type
     return None
 
+def get_option_format(device_type, module_id):
+    try:
+        # Search through nested dicts and see if 'format' key exists for the config option
+        format = [
+            DEVICE[device_type]['config'][config]['format'] for config in DEVICE[device_type]['config'].keys()
```

Comment:
Yes, I agree this would be better for long term maintenance. I don't want to refactor this code as part of this PR though, as it is not directly related to qos feature.

Reply:
Ok, then let is stay as is.